<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumorsConversionDialog</class>
 <widget class="QDialog" name="NumorsConversionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASCII to HDF5 numors converter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeView" name="treeView_inputFiles">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Diffractometer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_diffractometers"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_outputDirectory">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBar_conversion">
       <property name="styleSheet">
        <string notr="true">QProgressBar {
                                         border: 1px solid black;
                                         text-align: center;
											  color: white;
                                         padding: 1px;
                                         border-bottom-right-radius: 7px;
                                         border-bottom-left-radius: 7px;
                                         background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
                                         stop: 0 #eee,
                                         stop: 0.5 #ddd,
                                         stop: 1 #eee );
                                         width: 15px;
                                         }
                                         
                                         QProgressBar::chunk {
                                         background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
                                         stop: 0 #78d,
                                         stop: 1 #238 );
                                         border-bottom-right-radius: 7px;
                                         border-bottom-left-radius: 7px;
                                         border: 1px solid black;
                                         }</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_convert">
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
